pkg/plugin/framework/action_resolver.go

```diff
@@ -0,0 +1,278 @@
+/*
+Copyright the Velero Contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package framework
+
+import (
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/vmware-tanzu/velero/pkg/plugin/velero"
+	isv1 "github.com/vmware-tanzu/velero/pkg/plugin/velero/item_snapshotter/v1"
+
+	"github.com/vmware-tanzu/velero/pkg/discovery"
+	"github.com/vmware-tanzu/velero/pkg/util/collections"
+)
+
+/*
+Velero has a variety of Actions that can be executed on Kubernetes resources.  The Actions (BackupItemAction, RestoreItemAction
+and others) implement the Applicable interface which returns a ResourceSelector for the Action.  The ResourceSelector
+can specify namespaces, resource names and labels to include or exclude.  The ResourceSelector is resolved into lists
+of namespaces and resources present in the backup to be matched against.  These lists and the label selector are then used to
+decide whether or not the ResolvedAction should be used for a particular resource.
+*/
+
+type ResolvedAction interface {
```

Comment:
Could you update the comment to each interface like 
`// ResolvedAction ... ...`
Similar to the style of
https://github.com/vmware-tanzu/velero/blob/main/pkg/plugin/velero/backup_item_action.go#L26